src/main/java/in/pune/royforge/eledgerapi/controller/WalletController.java

```diff
@@ -34,4 +35,11 @@ public void createWallet(@RequestBody WalletTransaction walletTransaction) {
 	public WalletData getWallet(@PathVariable(value = "walletId") Long walletId) {
 		return walletEntityService.getWallet(walletId);
 	}
+
+	@RequestMapping(value = "/{lenderId}/{borrowId}", method = RequestMethod.GET)
+	@ResponseBody
+	public List<WalletData> getListOfWalletById(@PathVariable(value = "lenderId") String lenderId,
```

Comment:
change return type  List<WalletData> to WalletData because only single WalletData is available for this condition